components/board/client.go

```diff
@@ -193,10 +188,6 @@ func (dic *digitalInterruptClient) Value(ctx context.Context, extra map[string]i
 	return resp.Value, nil
 }
 
-func (dic *digitalInterruptClient) Tick(ctx context.Context, high bool, nanoseconds uint64) error {
-	panic(errUnimplemented)
```

Comment:
changes in this file unrelated to this change but noticed that this should've been cleaned up.